beacon-chain/sync/rpc_blob_sidecars_by_root.go

```diff
@@ -82,6 +82,24 @@ func (s *Service) blobSidecarByRootRPCHandler(ctx context.Context, msg interface
 	return nil
 }
 
+func (s *Service) blobSidecarsByRangeRPCHandler(ctx context.Context, msg interface{}, stream libp2pcore.Stream) error {
```

Comment:
note that the file says by root and this is by range, I don't feel strongly though